packages/form-field/src/FormField/FormField.tsx

```diff
@@ -46,8 +39,8 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
       state = FormFieldState.None,
       size = Size.Default,
       disabled = false,
-      errorMessage,
-      successMessage,
+      errorMessage = DEFAULT_MESSAGES.error,
+      successMessage = DEFAULT_MESSAGES.success,
```

Comment:
I previously thought we might want to customize a different default `errorMessage` or `successMessage` depending on the input component, but upon further reflection, I think it's fine to set this broadly at the form field level. Open to feedback if we'd prefer to define at the wrapping component level (text input, combobox, etc) since I'm also exporting `DEFAULT_MESSAGES` from the form field package

Reply:
Seems the designs all have the same messages. We can update the default messages per-component if we need to as well